requirements.txt

```diff
@@ -0,0 +1,30 @@
+Babel==2.11.0
```

Comment:
As it is required by python only, this file may not be added in the root folder of the repository